code-challenges.test.js

```diff
@@ -49,15 +78,55 @@ const randomNouns1 = ["streetlamp", "potato", "teeth", "conclusion", "nephew"]
 const randomNouns2 = ["temperature", "database", "chopsticks", "mango", "deduction"]
 // Expected output: ["Temperature", "Database", "Chopsticks", "Mango", "Deduction"]
 
+// psuedo code
+// create a funciton
+// input will be array of strings
+// .map to iterate
+//  capitalize the value of the first index of each word in the array
+// return results
+// output an array with capitalized words
+
+describe("capitalizeWords", () => {
+  it("takes in an array of words and returns an array with all the words capitalized.", () => {
+    expect(capitalizeWords(randomNouns1)).toEqual(["Streetlamp", "Potato", "Teeth", "Conclusion", "Nephew"])
+    expect(capitalizeWords(randomNouns2)).toEqual(["Temperature", "Database", "Chopsticks", "Mango", "Deduction"])
+  })
+})
+// ✕ takes in an array of words and returns an array with all the words capitalized. (1 ms)
+//
+// ● capitalizeWords › takes in an array of words and returns an array with all the words capitalized.
+
+
+
 
 // b) Create the function that makes the test pass.
 
+const capitalizeWords = (array) => {
+  let eachItem = array.map(value => value[0].toUpperCase() + value.substring(1))
+  return eachItem
+}
+
+console.log(capitalizeWords(randomNouns1));
+console.log(capitalizeWords(randomNouns2));
```

Comment:
Nice work on the code logic. As a next step you can return line 105 rather than creating a variable.